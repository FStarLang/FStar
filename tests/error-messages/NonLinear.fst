module NonLinear

[@@expect_failure [138]]
let test (l : list int) =
  match l with
  | x :: x :: [] -> 1
  | _ -> 0
