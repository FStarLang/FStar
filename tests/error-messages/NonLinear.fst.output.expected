>> Got issues: [
* Error 138 at NonLinear.fst(6,9-6,10):
  - Non-linear patterns are not permitted: `x` appears more than once in this pattern.

>>]
